dart/lib/src/sentry_client.dart

```diff
@@ -485,6 +485,75 @@ class SentryClient {
     );
   }
 
+  @internal
+  Future<void> captureLog(
+    SentryLog log, {
+    Scope? scope,
+  }) async {
+    if (!_options.enableLogs) {
+      return;
+    }
+
+    log.attributes['sentry.sdk.name'] = SentryLogAttribute.string(
+      _options.sdk.name,
+    );
+    log.attributes['sentry.sdk.version'] = SentryLogAttribute.string(
+      _options.sdk.version,
+    );
```

Comment:
do you know if these keys will be used multiple times? if yes we can consider putting them in a constants files or something.

we have a `constants.dart` in the dart package

Reply:
Checked python. They are only used once in the client. So it should be ok to just leave them here for now, and move to constants if we need them in other places.